### PR TITLE
update assets.json

### PR DESCRIPTION
**Pull request by wuzhong-papermoon**
Update information in assets.json

### CODE REVIEW

File: assets.json [hunk 1]
```diff
@@ -3,570 +3,476 @@
       {
         "name": "Moonriver",
```
Comment (albertov19) [inline assets.json]: Hey as for the name. I would indeed add something that references that this token is Composable Finance Moonriver IBC for example

File: assets.json [hunk 2]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "HydraDX",
         "assetID": "69606720909260275826784788104880799692",
-        "api-name": "hydradx",
-        "price": 0
+        "api-name": "hydradx"
       },
       {
         "name": "Celestia",
```
Comment (albertov19) [inline assets.json]: Composable Finance Celestia IBC

File: assets.json [hunk 3]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "Equilibrium Token",
         "assetID": "190590555344745888270686124937537713878",
-        "api-name": "equilibrium",
-        "price": 0
+        "api-name": "equilibrium"
       },
       {
         "name": "Inter Stable Token",
```
Comment (albertov19) [inline assets.json]: This is another Composable one right?

File: assets.json [hunk 4]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "Astar",
         "assetID": "224077081838586484055667086558292981199",
-        "api-name": "astar",
-        "price": 0
+        "api-name": "astar"
       },
       {
         "name": "Agoric",
```
Comment (albertov19) [inline assets.json]: Composoable

File: assets.json [hunk 5]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "Subsocial",
         "assetID": "89994634370519791027168048838578580624",
-        "api-name": "subsocial",
-        "price": 0
+        "api-name": "subsocial"
       },
       {
         "name": "Cosmos Hub",
```
Comment (albertov19) [inline assets.json]: Composable

File: assets.json [hunk 6]
```diff
@@ -3,570 +3,476 @@
...
         "assetID": "138280378441551394289980644963240827219",
-        "api-name": "cosmos",
-        "price": 0
+        "api-name": "cosmos"
       },
       {
         "name": "Picasso",
```
Comment (albertov19) [inline assets.json]: Composable

File: assets.json [hunk 7]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "BNC",
         "assetID": "165823357460190568952172802245839421906",
-        "api-name": "bifrost-native-coin",
-        "price": 0
+        "api-name": "bifrost-native-coin"
       },
       {
         "name": "PARA",
```
Comment (albertov19) [inline assets.json]:
```suggestion
        "name": "Parallel Finance",
```

File: assets.json [hunk 8]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "Interlay",
         "assetID": "101170542313601871197860408087030232491",
-        "api-name": "interlay",
-        "price": 0
+        "api-name": "interlay"
       },
       {
         "name": "BNC",
```
Comment (albertov19) [inline assets.json]:
```suggestion
        "name": "Bifrost",
```

File: assets.json [hunk 9]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "PARA",
         "assetID": "32615670524745285411807346420584982855",
-        "api-name": "parallel-finance",
-        "price": 0
+        "api-name": "parallel-finance"
       },
       {
         "name": "xcDOT",
```
Comment (albertov19) [inline assets.json]:
```suggestion
        "name": "Polkadot",
```

File: assets.json [hunk 10]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "Phala Token",
         "assetID": "189307976387032586987344677431204943363",
-        "api-name": "pha",
-        "price": 0
+        "api-name": "pha"
       },
       {
         "name": "aSEED",
```
Comment (albertov19) [inline assets.json]:
```suggestion
        "name": "Acala aSEED",
```

File: assets.json [hunk 11]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "Picasso",
         "assetID": "228510780171552721666262089780561563481",
-        "api-name": "picasso",
-        "price": 0
+        "api-name": "picasso"
       },
       {
         "name": "Acala Dollar",
```
Comment (albertov19) [inline assets.json]:
```suggestion
        "name": "Acala aSEED",
```

File: assets.json [hunk 12]
```diff
@@ -3,570 +3,476 @@
...
       },
       {
         "name": "xcBNC",
         "assetID": "319623561105283008236062145480775032445",
-        "api-name": "bifrost-native-coin",
-        "price": 0
+        "api-name": "bifrost-native-coin"
       }
     ],
     "moonbase": [
```
Comment (albertov19) [inline assets.json:256]: I would remove all Moonbase ones and just leave `xcUNIT` for testing

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looking good, left some comments

File: assets.json (hunk 1 above)
Comment (wuzhong-papermoon) [reply assets.json]: Does this work?
```suggestion
        "name": "Composable Finance Moonriver IBC",
```

File: assets.json (hunk 2 above)
Comment (wuzhong-papermoon) [reply assets.json]:
```suggestion
        "name": "Composable Finance Celestia IBC",
```

File: assets.json (hunk 3 above)
Comment (wuzhong-papermoon) [reply assets.json]: Yes!
```suggestion
        "name": "Composable Finance Inter Stable Token IBC",
```

File: assets.json (hunk 4 above)
Comment (wuzhong-papermoon) [reply assets.json]:
```suggestion
        "name": "Composable Finance Agoric IBC",
```

File: assets.json (hunk 5 above)
Comment (wuzhong-papermoon) [reply assets.json]:
```suggestion
        "name": "Composable Finance Cosmos Hub IBC",
```

File: assets.json (hunk 6 above)
Comment (wuzhong-papermoon) [reply assets.json]:
```suggestion
        "name": "Composable Finance Picasso IBC",
```

File: assets.json [hunk 13]
```diff
@@ -1,233 +1,479 @@
...
+      {
+        "name": "ICZ",
+        "assetID": "246731721631919099516624243100078522143",
+        "price": 0
+      },
+      {
+        "name": "OriginTrail Parachain Token",
+        "assetID": "238111524681612888331172110363070489924",
+        "api-name": "origintrail-parachain"
+      }
```
Comment (wuzhong-papermoon) [inline assets.json]:
```suggestion
      {
        "name": "xcUNIT",
        "assetID": "42259045809535163221576417993425387648",
        "price": 0
      }
```

File: assets.json [hunk 14]
```diff
@@ -1,233 +1,264 @@
...
+      },
+      {
+        "name": "LIT",
+        "assetID": "65216491554813189869575508812319036608",
+        "api-name": "litentry"
+      },
+      {
+        "name": "Crab Parachain Token",
+        "assetID": "173481220575862801646329923366065693029",
+        "price": 0.00099
```
Comment (wuzhong-papermoon) [inline assets.json]:
```suggestion
        "price": 0.0009651
```

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**